src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -85,6 +85,8 @@ class OTARequestor : public OTARequestorInterface
         kStartBDX,
     };
 
+    struct QueryImageRequest;
```

Comment:
should this be here?

Reply:
Yes, it's an argument of the method below.